examples/plot_kernel_ridge_regression.py

```diff
@@ -136,9 +137,9 @@
         test_time.append(time.time() - t0)
 
     plt.plot(sizes, train_time, 'o-', color="r" if name == "SVR" else "g",
-             label="%s (train)" % name)
+             markeredgecolor=(0, 0, 0), label="%s (train)" % name)
```

Comment:
> As I compared it with Dev version I noticed a minute difference so I thought it's better to add edgecolor though both look ok.

I know it is tough to give a general rule, but you need to exercise some judgement whether the plot looks a lot worse because of matplotlib v2. In this case the dev version is fine, so there is no need to change it.